src/Analyzers/MSTest.Analyzers/Resources.resx

```diff
@@ -191,6 +191,12 @@
   <data name="AssemblyInitializeShouldBeValidTitle" xml:space="preserve">
     <value>AssemblyInitialize methods should have valid layout</value>
   </data>
+  <data name="AssertionArgsShouldAvoidConditionalAccessMessageFormat" xml:space="preserve">
+    <value>Assert methods should not contain conditional access. First assert for null and then do the other assertion.</value>
+  </data>
+  <data name="AssertionArgsShouldAvoidConditionalAccessTitle" xml:space="preserve">
+    <value>Assertion arguments should avoid conditional access</value>
```

Comment:
What about `Avoid conditional access in assertions`? cc @nohwnd

Reply:
Updated